src/components/pages/rules/flyouts/ruleViewer/ruleViewer.js

```diff
@@ -128,6 +128,21 @@ export class RuleViewer extends Component {
           </Section.Content>
         </Section.Container>
 
+        {
+          rule.actions.length > 0 &&
+          <Section.Container collapsable={false}>
+            <Section.Content>
+              <div className="rule-action-title">{t('rules.flyouts.ruleEditor.actions.action')}</div>
+              <div className="rule-prop-label">{t('rules.flyouts.ruleEditor.actions.emailAddresses')}</div>
+              <div className="rule-prop-value">{rule.actions[0].parameters.recipients.join(", ")}</div>
```

Comment:
Why is only actions[0] displayed? What if there were more?

Reply:
right now we are only allowing 1 action. For simplicity since this is crawl scenario assuming 1 action